lib/backup/fscommon/fscommon.go

```diff
@@ -248,11 +248,11 @@ func removeEmptyDirsInternal(d *os.File) (bool, error) {
 	if dirEntries > 0 {
 		return false, nil
 	}
-	logger.Infof("removing empty dir %q", dir)
 	if hasFlock {
+		// root dir contains additional entries, so root lock won't be removed
 		flockFilepath := dir + "/flock.lock"
 		if err := os.Remove(flockFilepath); err != nil {
-			return false, fmt.Errorf("cannot remove %q: %w", flockFilepath, err)
+			return false, fmt.Errorf("cannot remove %q: %s", flockFilepath, err)
 		}
 	}
 	if err := os.Remove(dir); err != nil {
```

Comment:
```suggestion
	// Use os.RemoveAll() instead of os.Remove(), since the dir may contain ignored files such as flock.lock and restore-in-progress
	if err := os.RemoveAll(dir); err != nil {
```